samples/attestation_plugin/enclave/plugin1/plugin1.c

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <mbedtls/sha256.h>
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/safemath.h>
+#include <openenclave/enclave.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "plugin1.h"
+
+// This plugin emulates the functionality of `oe_get_report` and
+// `oe_verify_report`  for remote attestation reports through
+// the plugin interface.
+
+static int oe_on_register(
+    oe_quote_customization_plugin_context_t* plugin_context,
+    const void* config_data,
+    size_t config_data_size)
+{
+    OE_UNUSED(plugin_context);
+    OE_UNUSED(config_data);
+    OE_UNUSED(config_data_size);
+    return 0;
+}
+
+static int oe_on_unregister(
+    oe_quote_customization_plugin_context_t* plugin_context)
+{
+    OE_UNUSED(plugin_context);
+    return 0;
+}
+
+static int _calc_sha256(
+    const uint8_t* user_data,
+    size_t user_data_size,
+    uint8_t* output)
+{
+    mbedtls_sha256_context ctx;
+    mbedtls_sha256_init(&ctx);
+    int ret = -1;
+
+    if (mbedtls_sha256_starts_ret(&ctx, 0) != 0)
+        goto done;
+
+    if (mbedtls_sha256_update_ret(&ctx, user_data, user_data_size) != 0)
+        goto done;
+
+    if (mbedtls_sha256_finish_ret(&ctx, output) != 0)
+        goto done;
+
+    ret = 0;
+
+done:
+    mbedtls_sha256_free(&ctx);
+    return ret;
+}
+
+static int oe_get_evidence(
+    oe_quote_customization_plugin_context_t* plugin_context,
```

Comment:
another name without oe_ would be less confusing even it's already declared as "static"

Reply:
Sounds good.